src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/SqlDataReader.cs

```diff
@@ -27,7 +28,7 @@
 namespace Microsoft.Data.SqlClient
 {
     /// <include file='../../../../../../../doc/snippets/Microsoft.Data.SqlClient/SqlDataReader.xml' path='docs/members[@name="SqlDataReader"]/SqlDataReader/*' />
-    public class SqlDataReader : DbDataReader, IDataReader
+    public class SqlDataReader : DbDataReader, IDataReader, IDbColumnSchemaGenerator
```

Comment:
This change (as well as the additional method) to the public API surface needs to be replicated into the netfx ref project. Once you've done this, I think CI will compile. Visual Studio compiles using a project reference, CI might use a package reference (which introduces the ref project as a layer of indirection.)

Reply:
Thank you!

Reply:
I haven't seen it compile on the NuGet stage, but the other netfx targets seem to compile their tests properly. Good catch on the netcore ref project though, I didn't realise that this was missing from there too.